mappings/net/minecraft/block/pattern/BlockPattern.mapping

```diff
@@ -0,0 +1,61 @@
+CLASS none/asq net/minecraft/block/pattern/BlockPattern
+	CLASS none/asq$a BlockStateCacheLoader
+		FIELD a world Lnone/aiw;
+		FIELD b liveState Z
+		METHOD <init> (Lnone/aiw;Z)V
+			ARG 0 world
+			ARG 1 liveState
+		METHOD a load (Lnone/cn;)Lnone/asp;
+			ARG 0 pos
+	CLASS none/asq$b Result
+		FIELD a frontTopLeft Lnone/cn;
+		FIELD b forwards Lnone/cu;
+		FIELD c up Lnone/cu;
+		FIELD d cache Lcom/google/common/cache/LoadingCache;
+		FIELD e leftSize I
+		FIELD f downSize I
+		FIELD g forwardSize I
```

Comment:
What's wrong with `x/y/z` and `width`/`height`/`depth` or `x/y/zSize` or `sizeX/Y/Z` ?